webapp/channels/src/components/post/post_options.tsx

```diff
@@ -210,7 +210,7 @@ const PostOptions = (props: Props): JSX.Element => {
     let pluginItems: ReactNode = null;
     const pluginItemsVisible = usePluginVisibilityInSharedChannel(post.channel_id);
 
-    if ((!isEphemeral && !post.failed && !systemMessage) && hoverLocal && pluginItemsVisible) {
+    if ((!isEphemeral && !post.failed && !systemMessage && !isBurnOnReadPost) && hoverLocal && pluginItemsVisible) {
```

Comment:
Will this take care of Agents plugin actions too? It seems that Agents has a high level entry point and not in the same item for all other plugin actions. Asking as I'm not sure if it's another pluggable area that we may need to consider